Project/Homebrew/qctools.pro

```diff
@@ -0,0 +1,95 @@
+QT += core gui widgets svg printsupport
+
+TARGET = QCTools
+TEMPLATE = app
+
+QT_CONFIG -= no-pkg-config
+
+include ( $$system(brew --prefix qwt)/features/qwt.prf )

```

Comment:
I just used this .project to build for Linux. The only line that gives any problem is this one which is dependent on homebrew. Should probably add some heuristics here checking for homebrew if on mac, and common paths for qwt prf's files and if not found quit with error message.